assets/locales/en.json

```
@@ -116,5 +110,26 @@
   "itemsFailedNotification": {
     "title": "Failed to add items",
     "message": "Could not add your items to your node."
+  },
+  "updateAvailableDialog": {
+    "title": "Update available",
+    "message": "A new version { version } is available. The download will begin shortly in background."
```

```suggestion
    "message": "A new version ({ version }) of IPFS Desktop is available. The download will begin shortly in the background."
```
Also a question: what if the user doesn't want to update? Do we need to give them an option to cancel?

---

Hmmm... If they'd cancel, the download would end up getting downloaded after 12 hours because we check every 12 hours for a new update. Maybe, in the future, we can introduce an option to disable auto-update.

---

Ah, got it. Thanks 😊